wordpress-mobile/gutenberg-mobile
# Add border on Title when focused

Fix https://github.com/wordpress-mobile/gutenberg-mobile/issues/540
Fix https://github.com/wordpress-mobile/gutenberg-mobile/issues/551

This change adds borders to the Title when focused in order to match other blocks.
And also fixes a slight shift happening on Android when we focus to a block.

gutenberg PR needs to be merged first: https://github.com/WordPress/gutenberg/pull/13970

![android-title-border](https://user-images.githubusercontent.com/5032900/53090362-3a02d300-3520-11e9-8b0f-97a27a110d63.gif)

![ios-title-border](https://user-images.githubusercontent.com/5032900/53090388-53a41a80-3520-11e9-9e6f-3aedcf037397.gif)

**To Test**

Testing prerequisites

For WPiOS

Checkout the PRs branch to any arbitrary folder and cd .. to it
run yarn install, yarn start
Open XCode WPiOS on the latest develop
Clean build folder on Xcode, and then run the app

For WPAndroid

open grade.properties at WordPress-Android folder
add wp.BUILD_GUTENBERG_FROM_SOURCE = true to grade.properties
checkout the PRs branch in the subrepo of WordPress-Android repo
cd to WordPress-Android/libs/gutenberg-mobile
run yarn install, yarn start
yarn wpandroid on a separate terminal in the same directory

Test Steps:

WPiOS:

- Open a post and tap the title
- Verify you see focus indicating borders around the title just like the other blocks
- Verify there's no slight shifting on portrait mode when we focus on a block because there should be no left right borders

iPhone X and iPad special case:
- Verify that you do see the left right borders when there's empty space around the block like there's on iPhone X landscape mode, and iPad portrait and landscape modes.

WPAndroid:

- Open a post and tap the title
- Verify you see focus indicating borders around the title just like the other blocks
- Verify that the slight shift that is described here is fixed also: https://github.com/wordpress-mobile/gutenberg-mobile/issues/551
